### DISCUSSION TITLE
Is there any way to create a terminal script?

### DISCUSSION THREAD

**Discussion by TheMissingPort** [UltimaHoarder/UltimaScraper#1020] [General]
Like a macro that I press that will automatically change directory and enter python3 start_ofd.py so I don't have to do it manually every time? 

Thanks

**Comment by UltimaHoarder** [comment]
You can load main_datascraper.py into your own custom script with a custom json_config.

**Comment by LunarPenguin** [comment]
You could set `"auto_site_choice"`, "`auto_profile_choice"`, `"auto_scrape_names": false`, `"auto_choice"` and `"download_directories"` the script should then auto run when `start_ofd.py` is executed which could be done by a cronjob
